apps/custom-config/package.json

```diff
@@ -7,6 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "manifest": {
+    "daemon": true,
```

Comment:
daemon app should be considered strictly, why not receiving the message at component-level module, and use openUrl to notify to application?

Reply:
We will refactor this to component, so we can do this at that time.